charts/spire/charts/spire-agent/values.yaml

```diff
@@ -165,6 +165,8 @@ workloadAttestors:
     ## @param workloadAttestors.unix.enabled Enables the Unix workload attestor
     enabled: false
   k8s:
+    ## @param workloadAttestors.k8s.enabled Enables the Kubernetes workload attestor
+    enabled: true
```

Comment:
See previous comment. Is there ever a situation where you don't want k8s attestation in a Kubernetes deployment?